### DISCUSSION TITLE
Downloading dandiset into certain directory/external drive?

### DISCUSSION THREAD

**Discussion by srilakshmip03** [dandi/helpdesk#138] [Q&A]
Hello! I've just started using DANDI and I was wondering how to specify the directory that a dataset downloads to? I'm downloading one thats 600+ GB, so I was hoping to load the dataset directly onto an external drive. Is this possible?

Thanks in advance!

**Comment by yarikoptic** [comment]
Are you using dandi CLI? Then just invoke the process from the directory where you want it under, since it downloads under current directory.

And there is -o option to specify folder, see https://dandi.readthedocs.io/en/latest/cmdline/download.html
